### PR TITLE
chore(security): Fix OCI Manifest Type Confusion vulnerability 

### PR DESCRIPTION
**Pull request by josvazg**
Refs https://github.com/advisories/GHSA-qq97-vm5h-rrhg

Note: Github's dependabot integration does not work.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -23,7 +23,7 @@ require (
 	github.com/containerd/stargz-snapshotter/estargz v0.4.1 // indirect
 	github.com/cyphar/filepath-securejoin v0.2.2 // indirect
 	github.com/docker/cli v20.10.5+incompatible // indirect
-	github.com/docker/distribution v2.7.1+incompatible // indirect
+	github.com/docker/distribution v2.8.0+incompatible // indirect
```
Comment (migmartri) [inline go.mod:26]: Why didn't we update to the next patch version? 2.8.1?

Comment (josvazg) [reply go.mod:26]: no particular reason, I usually take the next version I know exists. But if there is a 2.8.1 I guess it will be fine too.

Comment (josvazg) [reply go.mod]: bumped to v2.8.1

**Review by petewall — APPROVED** [review]
Looks good!

**Pull request merged**